### PR TITLE
Address Issue 101 and clarify header field descriptions

### PR DESCRIPTION
**Pull request by abellgithub**
Note that I was unsure about requiring scale/offset values to be 0 if there are no points in the file. I don't think that restriction is necessary, but the previous language was vague, although setting min/max to 0 when there are no points seems just as ill-defined with the previous language.

### CODE REVIEW

**Comment by esilvia** [comment]
Tagging #101

File: source/02.04_header.sub [hunk 1]
```diff
@@ -104,7 +99,7 @@ as a quick look initial determination of file type.
 
 This field should be set to a value from 0 to 65,535. A value of zero is
 interpreted to mean that an ID has not been assigned, which is the norm for a
-LAS file resulting from an aggregation of multiple independent sources (e.g., a
+file resulting from an aggregation of multiple independent sources (e.g., a
```
Comment (esilvia) [inline source/02.04_header.sub:102]: Why did you remove "LAS" in multiple places? Does it handle a certain use case?

**Comment by esilvia** [comment]
It might seem inconsistent, but I don't think it makes sense to require a null scale/offset. Scale/offset are prescriptive and tend to be consistent within a set of LAS files, while min/max are descriptive of the specific LAS file. I'll try to finish reviewing this by tomorrow.

File: source/02.04_header.sub (hunk 1 above)
Comment (abellgithub) [reply source/02.04_header.sub:102]: This is a specification for LAS. Repeating the acronym "LAS" adds nothing. It's also more consistent.

**Review by esilvia — CHANGES_REQUESTED** [review]
I've tried to go over these changes a few times now, and unfortunately I think that too many editorial changes unrelated to #101 are included in this PR for me to accept it. I'm generally okay with the changes related to #101 but I think some of the other clarifying changes need further discussion before I can accept them. Since the changes are all on the same file it's non-trivial to split them apart.

Are you able to split the editorial changes unrelated to the zero-header values into a separate PR and issue so that they can be discussed separately from the changes related to #101? I can attempt to do so, but maybe you'd rather do it yourself? I'd like to include #101 in R16.

**Comment by abellgithub** [comment]
I'm way past caring.

**Comment by esilvia** [comment]
Very well... sorry I took so long. I did my best to capture the portion of your work relevant to 101 in PR #108
